tcms/rpc/tests/test_testcase.py

```diff
@@ -541,6 +644,31 @@ def test_add_comment_with_pk_as_int(self):
         self.assertEqual("Hello World!", created_comment['comment'])
 
 
+class TestRemoveCommentPermissions(APIPermissionsTestCase):
+    permission_label = 'django_comments.delete_comment'
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.case = TestCaseFactory()
+        self.comment_1 = comments.add_comment([self.case], 'First one', self.tester)[0]
+        self.comment_2 = comments.add_comment([self.case], 'Second one', self.tester)[0]
+        self.comment_3 = comments.add_comment([self.case], 'Third one', self.tester)[0]
+
+    def verify_api_with_permission(self):
+        # Remove a specific comment
+        self.rpc_client.TestCase.remove_comment(self.case.pk, self.comment_1.pk)
+        self.assertEqual(len(comments.get_comments(self.case)), 2)
+
+        # Remove all comments
+        self.rpc_client.TestCase.remove_comment(self.case.pk)
+        self.assertEqual(len(comments.get_comments(self.case)), 0)
+
+    def verify_api_without_permission(self):
+        with self.assertRaisesRegex(ProtocolError, '403 Forbidden'):
+            self.rpc_client.TestCase.remove_comment(self.case.pk)
```

Comment:
Missing scenario with invalid TC ID which asserts we get a DoesNotExist exception. See above.